xls/dslx/bytecode/bytecode_interpreter_test.cc

```diff
@@ -1519,6 +1519,78 @@ fn main() {
   EXPECT_TRUE(value.IsUnit());
 }
 
+// GH 981
+// Assert_eq type deduction didn't fail in
+// the first iteration of the next function.
+// This test requires 2 iterations of next function,
```

Comment:
let's add the command inline next to the `send_if` for better context.

Reply:
Is it better now?